base/range.jl

```diff
@@ -367,6 +370,26 @@ julia> LinRange(1.5, 5.5, 9)
 9-element LinRange{Float64}:
  1.5,2.0,2.5,3.0,3.5,4.0,4.5,5.0,5.5
 ```
+
+Compared to [`range`](@ref), `LinRange` should have less overhead but won't try to correct
+for floating point errors:
+```julia
+julia> collect(-0.1:0.1:0.3)
```

Comment:
```suggestion
julia> collect(range(-0.1, 0.3, length=5))
```
Less comparing apples to oranges.